lib/ui/checkbox.tsx

```diff
@@ -21,7 +23,11 @@ const Checkbox = React.forwardRef<
       onCheckedChange={props.onCheckedChange}
     >
       <CheckboxPrimitive.Indicator className="flex items-center justify-center text-current">
-        <Check className="h-3 w-3" />
+        {partial ? (
```

Comment:
there is an `indeterminate` html attribute ([mdn](https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement/indeterminate)) for this purpose. Let's use it